maintenance/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
```

Comment:
I went back and forth on that, too, and ultimately decided to include it because we don't know where this maintenance page will be hosted yet - i.e. it may not be in a container that will go through all the build steps. I was thinking a super low friction way of serving it up would be to set up a Github Pages project for it, in which case the actual file will need to be in version control.